guides/command_objects.md

```diff
@@ -6,7 +6,7 @@
 
 ## Conventions
 
-* Commands go under the `app/commands` directory.
+* Commands go under the `app/commands` directory if it doesn't related to any service. If it's a part of some service it can be under `app/services/some_service` directory (e.g. `GoogleService::RefreshUserDataCommand`).
```

Comment:
```suggestion
* Commands go under the `app/commands` directory. In case when a command is a part of a module, consider placing it into the directory structure suggested by [Service Modules](guides/service_modules.md).
```